docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+| scRNA-seq (non-UMI, e.g. SS2)         | Values MUST be one of read counts (e.g. FeatureCounts) or  estimated fragments (e.g. output of RSEM).                                         | REQUIRED        |                                                                                                    | STRONGLY RECOMMENDED  | OPTIONAL     |
+| Accessibility (e.g. ATAC-seq, mC-seq) |                                                                                                                                               | NOT REQUIRED    | Values MUST correspond to gene features (e.g. HGNC symbols if human)                               | REQUIRED              | OPTIONAL     |
 
 ### Integration Metadata
 
-To support data integration, each cell must have the following metadata:
+#### Cell Metadata
+
+To support data integration, each cell MUST have the following metadata:
 
 **Field name**|**Constraints**
 :--|:--
-tissue|string (see note below regarding cell culture and organoids)
+tissue|string. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay|string
 disease|string
 cell\_type|string
 sex|"male", "female", "mixed", "unknown", or "other"
 ethnicity|string, "na" if non-human, "unknown" if not available
 development\_stage|string, "unknown" if not available
 
-The `tissue` field must be appended with " (cell culture)" or " (organoid)" if appropriate. Also, if the source of cells is cell culture or organiod, the
-`cell_type` field can be left empty.
-In addition to these free text fields (except sex), each cell must also have ontology values:
+In addition to these free text fields (except sex), each cell MUST also have ontology annotations:
+
+Ontology terms MUST use [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning they are a CURIE
+where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
 
 **Field name**|**Constraints**
 :--|:--
-tissue\_ontology\_term\_id|UBERON term
+tissue\_ontology\_term\_id|UBERON term. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay\_ontology\_term\_id|EFO term
 disease\_ontology\_term\_id|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
 cell\_type\_ontology\_term\_id|CL term
 ethnicity\_ontology\_term\_id|HANCESTRO term, "na" if non-human
 development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:0000399 otherwise
 
-The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
+cellxgene requires ontology terms to enable search, comparison, and integration of data. When no appropriate ontology value is available, then the most
+precise accurate term MUST be used. For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL
```

Comment:
I see a line break between "most" and "precise" with the markdown is rendered.

Reply:
Addressed in #52